Hunted Mobile/Hunted Mobile/Hunted Mobile/Model/Location.cs

```diff
@@ -13,26 +13,14 @@ public class Location {
         /// <summary>
         /// Horizontal map coordinate
         /// </summary>
-        public double Latitude { get; set; }
+        public double Latitude { get; set; } = 5.000000;
 
         /// <summary>
         /// Vertical map coordinate
         /// </summary>
-        public double Longitude { get; set; }
+        public double Longitude { get; set; } = 51.000000;
```

Comment:
Ik denk niet dat de `Longitude` en `Latitude` standaard waardes nodig hebben, aangezien die altijd worden opgehaald.

Reply:
Ik denk het ook niet, maar deze waardes stonden in een Repository die ik refactored had, dus dacht waarom niet houden.